jwql/tests/test_pipeline_tools.py

```diff
@@ -56,29 +56,32 @@ def test_completed_pipeline_steps():
                                   ('jump', True),
                                   ('rate', True)])
 
-    assert completed_steps == true_completed
+    assert dict(completed_steps) == dict(true_completed)
```

Comment:
Just want to double-check - @bourque did you mean for the `test_completed_pipeline_steps()` assertion to also be modified in this PR?

Reply:
@laurenmarietta Whoops, nice catch!  I'll make the update.